backend/src/main/java/com/redhat/cloud/notifications/routers/ValidationEndpoint.java

```diff
@@ -0,0 +1,45 @@
+package com.redhat.cloud.notifications.routers;
+
+import com.redhat.cloud.notifications.db.ApplicationResources;
+import com.redhat.cloud.notifications.models.EventType;
+import io.smallrye.mutiny.Uni;
+import org.jboss.resteasy.reactive.RestQuery;
+
+import javax.inject.Inject;
+import javax.persistence.NoResultException;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import static com.redhat.cloud.notifications.Constants.INTERNAL;
+
+@Path(INTERNAL + "/validation")
+public class ValidationEndpoint {
+
+    private static final String EVENT_TYPE_NOT_FOUND_MSG = "No event type found for [bundleName=%s, applicationName=%s, eventTypeName=%s]";
+
+    @Inject
+    ApplicationResources appResources;
+
+    @GET
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/baet")
+    public Uni<Response> validate(@RestQuery String bundle, @RestQuery String application, @RestQuery String eventType) {
+        return appResources.getEventType(bundle, application, eventType)
+                .onItem()
+                .transform(this::convertToOkayResponse)
```

Comment:
Optional change (the current version works):
```suggestion
                .replaceWith(() -> Response.ok().build())
```

Reply:
That would also require to remove the method below.